src/components/CollectionItemCard/LanguageDisplayText.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Box } from "@nypl/design-system-react-components";
+
+const LanguageDisplayText: React.FC<{
+  language: string;
+}> = ({ language }) => {
+  return (
+    <Box as="p">
```

Comment:
Wasn't sure about the accessibility rules for putting text directly in a div, so I added this `as` prop, but it might change the display. I'll leave it up to you @jackiequach if this should be a p tag or just a div.

Reply:
I think this is better for accessibility, but you will need to add `margin="0"` to fix the spacing change.